domain/domain.go

```diff
@@ -79,6 +79,7 @@ type Domain struct {
 	statsUpdating        sync2.AtomicInt32
 	cancel               context.CancelFunc
 	indexUsageSyncLease  time.Duration
+	indexUsageSyncing    sync2.AtomicInt32
```

Comment:
This variable and the function `SetIndexUsageSyncing` seems unnecessary, since it will not change after the TiDB starts.